### DISCUSSION TITLE
Where do bug reports currently go?

### DISCUSSION THREAD

**Discussion by AshleyPikachu** [PintaProject/Pinta#1174] [Q&A] [answered]
Hi, im a bit confused on where I am meant to post bug reports regarding some effects for pinta.

Thanks

**Comment by cameronwhite** [comment, marked as answer]
Bug reports can be filed at https://github.com/PintaProject/Pinta/issues.

The old bug tracker on Launchpad is still accessible though, since older versions of Pinta point there and existing issues haven't been moved over here
